src/components/detail/detail.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, text } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import Detail from './detail.js';
+
+storiesOf('Detail', module)
+  .add('default', () => {
+    const icon = select('icon', [null, ...OptionsHelper.icons], null);
+    const footnote = text('footnote', 'This detail may require a footnote.');
+    const children = text('children', 'An example of a detail.');
+
+    return (
```

Comment:
Redundant jsdoc info for prop types:

![image](https://user-images.githubusercontent.com/26925206/53880647-6b43ce80-4009-11e9-91ca-280596769dfb.png)

Reply:
thanks, done